backend/handlers/handlers.go

```diff
@@ -1058,6 +1069,29 @@ func (handlers *Handlers) getExchangesByRegion(r *http.Request) interface{} {
 	return exchanges.ListExchangesByRegion(acct, handlers.backend.HTTPClient())
 }
 
+func (handlers *Handlers) getBitsuranceLookup(r *http.Request) interface{} {
+	type errorResult struct {
+		Error string `json:"error"`
+	}
+	insuredAccounts, err := handlers.backend.LookupInsuredAccounts(r.URL.Query().Get("code"))
+	if err != nil {
+		handlers.log.Error(err)
+		return errorResult{Error: err.Error()}
+	}
+
+	return insuredAccounts
```

Comment:
Did this way because most of the GET endpoint actually just returns data if everything goes well. But yeah, a response struct is much better, will do it like that. Would be nice to refactor the other endpoints as well in future PRs